scripts/Tools/Makefile

```diff
@@ -640,6 +613,9 @@ ifeq ($(USE_CXX), true)
   endif
 endif
 
+# Remove arch flag if it exists
```

Comment:
why? seems pretty obscure

Reply:
it breaks the linking of the model (e3sm.exe).